core/parameters/res/runtime_configs/72.yaml

```diff
@@ -1 +1,7 @@
-main_storage_proof_size_soft_limit: {old: 3_000_000, new: 4_000_000}
\ No newline at end of file
+main_storage_proof_size_soft_limit: {old: 3_000_000, new: 4_000_000}
+wasm_storage_has_key_base: { old: { gas: 54_039_896_625, compute: 200_000_000_000 }, new: { gas: 54_039_896_625, compute: 8_000_000_000 } }
+wasm_storage_has_key_byte: { old: 30_790_845, new: { gas: 30_790_845, compute: 9_000_000 } }
```

Comment:
Good catch, thank you! This is now fixed :)
(For people who were not in our DMs, I forgot to include the ReadTTN overestimated costs in `has_key_base`, this is now fixed by making it 158Ggas)

Reply:
Uhhh wait you wrote `has_key_byte` here instead of `has_key_base`, but our zulip exchanges had mentioned `has_key_base`. Is this some other problem that I'm not understanding, or is it just a typo?

Reply:
Firstly, I think we do need at least large/small distinction at least for `base` and `value_byte` costs. Because both the base latency and the throughput are different between on-disk and in-memory trie.

Also, you mentioned compute-only. While we could do it that way, it would make it harder to eventually, maybe, reduce gas costs of memtrie reads later in the future.

As for `small_read_key_byte`, I agree with you that we could probably do without it. However, I do think that it makes sense to add it, for consistency: if we had two `small_read` costs but not the third one, it would likely be surprising to future readers of the code. Considering having one additional cost is not a lot of lines of code, I feel like having it reduces tech debt by making the costs more straightforward.

IOW, I think it's better to semantically have two sets of three costs, one "small reads" and one "large reads"; than it'd be to have "read base, read key byte, read value byte, read overcost for large value base, read overcost for large value per value byte" costs.

Does that make sense?

Reply:
Makes sense, I just checked the raw data from the estimator, and it seems reasonable enough to set both at 10Mgas, there's a small performance difference but not that noticeable. Adjusting the numbers.